### ISSUE TITLE
Add "rad" and "deg" to Units

### ISSUE THREAD

**Issue by MuellerSeb** [mapado/haversine#31]
It would be nice to have the possibility to also return the distance in radians or degrees.

**Comment by jdeniau** [comment]
Hi, my math is far in my memory, but if `rad` and `deg` are angle units, so how do you convert a distance to an angle ? 🤔

**Comment by jdeniau** [comment]
To rephrase what I said : the input might be in rad or deg, but the output is actually a distance, 

If I look at the scikit learn implementation of haversine that you referenced in #30 their output is also in kilometers:

scikitlearn:
```py
>>> from sklearn.metrics.pairwise import haversine_distances
>>> from math import radians
>>> bsas = [-34.83333, -58.5166646]
>>> paris = [49.0083899664, 2.53844117956]
>>> bsas_in_radians = [radians(_) for _ in bsas]
>>> paris_in_radians = [radians(_) for _ in paris]
>>> result = haversine_distances([bsas_in_radians, paris_in_radians])
>>> result * 6371000/1000  # multiply by Earth radius to get kilometers
array([[    0.        , 11099.54035582],
       [11099.54035582,     0.        ]])
```

haversine:
```py
>>> bsas = (-34.83333, -58.5166646)
>>> paris = (49.0083899664, 2.53844117956)
>>> haversine(bsas, paris)
11099.555687157774
```

**Comment by MuellerSeb** [comment]
An angle in radians is simply the distance on the unit-sphere. So to get the result in rad, you simply don't multiply with the earth radius. The dependence between angle and distance in linear (that is why the circumference of a circle is r * 2pi (where 2pi is a 360 deg angle))

**Comment by jdeniau** [comment]
Fixed by #40
